scripts/fetch-docs.js

```diff
@@ -93,9 +98,20 @@ const fetchPages = async () => {
     // NOTE: this should be returned in the JSON payload
     const type = input.find(({ ids }) => ids.includes(doc.docId)).type;
     const frontmatter = getFrontmatter(type, doc);
+    const turndownService = new TurndownService({
+      headingStyle: 'atx',
+    });
+    turndownService.addRule('codeBlocks', {
+      filter: ['pre'],
+      replacement: function (content) {
+        // return '```' + '\n' + content + '```' + '\n';
+        return `~~~\n${content}\n~~~\n`;
+      },
+    });
+    turndownService.keep(['table']);
 
-    // Step 5: add content to file
```

Comment:
I think this commend is helpful to track the process.

Reply:
Adding back now